benches/nexmark/generator/mod.rs

```diff
@@ -2,5 +2,13 @@
 //!
 //! Based on the equivalent [Nexmark Flink generator API](https://github.com/nexmark/nexmark/blob/v0.2.0/nexmark-flink/src/main/java/com/github/nexmark/flink/generator).
 
+use crate::config::Config;
+use rand::Rng;
+
 mod people;
 mod strings;
+
+pub struct NexmarkGenerator<R: Rng + ?Sized> {
```

Comment:
Just from https://docs.rs/rand/latest/rand/trait.Rng.html#generic-usage : 

> The `+ ?Sized` un-bounding allows functions to be called directly on type-erased references; i.e. `foo(r)` where `r: &mut dyn RngCore`. Without this it would be necessary to write `foo(&mut r)`.

Maybe you mean it's not necessary now for the generic struct field? Not sure. I can remove it if you prefer.

Reply:
Yah, it's just an unneeded bound in this situation

Reply:
Ah - k. Removed and rebased on the target again after changes there. Txs.